higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -158,11 +158,16 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
 {% compress js inline %}
   <script type="text/javascript">
   "use strict";
+
   let universities = JSON.parse(document.getElementById('universities-data').textContent),
-  campuses = JSON.parse(document.getElementById('campus-data').textContent);
+  campuses = JSON.parse(document.getElementById('campus-data').textContent),
+  other_university = "{{ other_university }}";
```

Comment:
This is going to cause issues with the caching that `{% compress js inline %}` does.

Either, this needs to be a separate data element, or preferably, since we already have this data in this page, we look through the list of universities, and find the one whose label matches `other`.